package.json

```diff
@@ -89,10 +82,6 @@
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11"
   },
-  "collective": {
```

Comment:
Any reason this part was removed?

Reply:
Same reason I changed the author, will put it back.